### PR TITLE
Add nanoFramework

### PR DESCRIPTION
**Pull request by Ellerbach**
Running a first PR to get the main structure for .NET nanoFramework articles.
So far, here is the status:

* 2022-07-01: finalized and good for deep review
* 2022-07-08: finalized and good for deep review
* 2022-07-15: finalized and good for deep review
* 2022-07-22: in process but can be merged and then improved
* 2022-07-29: in process but can be merged and then improved

Questions:

* How to properly link the images in the static part? happy to learn about this
* How to have multiple authors, see the headers? not sure how to make this
* ok to merge even non finalized articles and then run another PR?
* what would be the best process all up to move forward?

### CODE REVIEW

File: docs/blog/2022-07-15-nanoframework-networking-azure.md [hunk 1]
```diff
@@ -0,0 +1,1067 @@
...
+    // Start the server.
+    server.Start();
+
+    Thread.Sleep(Timeout.Infinite);
+}
+```
+
+> IMPORTANT: because the certificate above is not issued by a Certificate Authority it won't be recognized as a valid certificate. If you want to access the nanoFramework device with your browser, for example, you'll have to add the (CRT file)[WebServer.Sample\webserver-cert.crt] as a trusted one. On Windows, you just have to double click on the CRT file and then click "Install Certificate...".
+
+You can of course use the routes as defined earlier. Both will work, event or route with the notion of controller.
```
Comment (gloveboxes) [inline docs/blog/2022-07-15-nanoframework-networking-azure.md]: This needs some clarification

**Review by gloveboxes — COMMENTED** [review]
Comments

File: docs/blog/2022-07-15-nanoframework-networking-azure.md (hunk 1 above)
Comment (Ellerbach) [reply docs/blog/2022-07-15-nanoframework-networking-azure.md]: I rewrote fully the sentence, you can check it, it's pushed on the branch

**Review by gloveboxes — APPROVED** [review]

**Review by gloveboxes — APPROVED** [review]
Happy with changes

**Comment by jimbobbennett** [comment]
Hey @Ellerbach. I'm working through these manually copying the code in. There's a few tweaks needed to the head matter, and ideally some of the image names should be clearer. I'm learning all this as I go along, hence why right now it's easier for me to manually merge.

We were hoping to have a contribution guide by now, but we don't.

I'll carry on manually merging these in and close this once done.

**Comment by Ellerbach** [comment]
> I'll carry on manually merging these in and close this once done.

OK! Thanks for the help. I guess the only article that will get more love and content is the one related to native build. And we have time for that one.

Once the contribution guide will be here, it will indeed be easier :-)

**Comment by Ellerbach** [comment]
@jimbobbennett any public URL to see the content as we are July 1rst? :-)

**Comment by jimbobbennett** [comment]
@Ellerbach https://julyot.dev

**Comment by jimbobbennett** [comment]
Closing this as the changes have been manually merged. These posts will be at:

https://julyot.dev/blog/01-nano-framework (already live)
https://julyot.dev/blog/08-nano-framework (going live 8th July)
https://julyot.dev/blog/15-nano-framework (going live 15th July)
https://julyot.dev/blog/22-nano-framework (going live 22nd July)
https://julyot.dev/blog/29-nano-framework (going live 29th July)

**Pull request closed without merging**